beacon-chain/operations/attestations/forkchoice/forkchoice.go

```diff
@@ -0,0 +1,87 @@
+package forkchoice
+
+import (
+	"sync"
+
+	"github.com/pkg/errors"
+	ethpb "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1/attestation"
+)
+
+// Attestations --
+type Attestations struct {
```

Comment:
I created a new type to hold forkchoice attestations to be able to reuse it between the old and new pools